changelogs/fragments/1241-vmware_vm_info-add_the_posibility_to_get_only_one_vm_by_name.yaml

```diff
@@ -0,0 +1,3 @@
+breaking_changes:
```

Comment:
This is not a breaking change, but a new feature. Can you use `minor_changes` instead.

Reply:
Next time i will use minor_changes :)

Reply:
No worries, it's just the we cannot release a breaking change. Can you push a PR to adjust the field? We will approave immediately. c.f @mariolenz 

BTW, thank you for the contribution :-).

Reply:
I chagned it :)
https://github.com/ansible-collections/community.vmware/pull/1271

Reply:
Sorry, my mistake. I didn't spot this when I've reviewed this PR :-/